python/MeshData.sip

```diff
@@ -24,12 +24,12 @@ class MeshData
 public:
     MeshData();
     virtual ~MeshData();
-    PyObject* getFacesAsBytes();
-    PyObject* getVerticesAsBytes();
+    std::vector<char> getVerticesAsBytes();
+    std::vector<char> getFacesAsBytes();
 
-    PyObject* getFlatVerticesAsBytes();
+    std::vector<char> getFlatVerticesAsBytes();
 
-    void setVerticesFromBytes(PyObject* py_bytes);
+    void setVerticesFromBytes(const std::vector<char> &data);
```

Comment:
Codestyle: & is part of the type, so this should be `const std::vector<char>& data`.